RELEASING.md

```diff
@@ -0,0 +1,18 @@
+# Release To The Google Play Store
+
+## Setup
+
+Run the setup script:
+
+```
+> bin/setup
+```
+
+## Releasing the app
+
+```
+> fastlane android release
```

Comment:
It looks like `>` is standard `PS2` but `PS1` is `$` on bash and `%` on zsh?

http://superuser.com/questions/57575/what-is-the-origin-of-the-unix-dollar-prompt
http://zsh.sourceforge.net/Guide/zshguide02.html#l19

I feel like we don't have a standard prompt style on the blog or for docs. @JoelQ or @gabebw or @mike-burns or @jferris might know. I think I've tended to not put the prompt to make it more copy-pasteable without accidentally copying the prompt. Might even pair with text beforehand like "From your prompt:" or "From your shell:"